elide-model-config/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydratorTest.java

```diff
@@ -275,6 +275,183 @@
             + "\n"
             + "}\n";
 
+    private static final String VALID_TABLE_GRAIN_JAVA_NAME = "PlayerStatsGrain";
+    private static final String VALID_TABLE_GRAIN_CLASS = "/*\n"
```

Comment:
Do we need to create such a complex model to test this?  THere is a lot of things unrelated to the test in this model.